extensions/CocoStudio/GUI/UIWidgets/ScrollWidget/UIScrollView.cpp

```diff
@@ -1586,7 +1606,9 @@ void ScrollView::interceptTouchEvent(int handleState, Widget *sender, const CCPo
 
 void ScrollView::checkChildInfo(int handleState,Widget* sender,const CCPoint &touchPoint)
 {
-    interceptTouchEvent(handleState, sender, touchPoint);
+	if (_enabled && _touchEnabled) {
```

Comment:
All the indentation are wrong here... please change to 4 spaces instead of a tab.
Thanks.